src/assets/netteForms.js

```diff
@@ -448,13 +448,14 @@
 			} catch (e) {} // eslint-disable-line no-empty
 		},
 
-		pattern: function(elem, arg, val) {
+		pattern: function(elem, arg, val, value, modifiers) {
```

Comment:
I think that argument `caseInsensitive` is better than `modifiers`.

Reply:
Ok. The same for the PHP bit then?

```php
pub fun xxx(..., bool $caseInsensitive = false) {
    $modifiers = 'u' . ($caseInsensitive ? 'i' : '');
}
```

Reply:
Yes